.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -88,7 +88,7 @@ body:
       description: |
         Please provide more information on your .NET configuration:
           * Which version of Fluent Assertions are you using?
-          * Which .NET runtime and version are you targeting? E.g. .NET framework 4.6.1 or .NET Core 2.1.
+          * Which .NET runtime and version are you targeting? E.g. .NET framework 4.6.1 or .NET 7.0.
```

Comment:
Though this is not wrong technically, it does not align with several other places (.NET 6.0).